### ISSUE TITLE
Path resolver support

### ISSUE THREAD

**Issue by justindarner** [ionutvmi/path-autocomplete#30]
It would be great if we get support for the following setting:
```
"path-autocomplete.pathMappings": {
    "test": "${workspace}/src/Actions/test", // alias for /test
    "": "${workspace}/src" // the root folder is now /src
}
```

The the following would autocomplete:
`import Grid from 'components/Grid';`

Grid would be located at: ${workspace}/src/componets/Grid

Note import Path Resolver
https://moduscreate.com/blog/es6-es2015-import-no-relative-path-webpack/

**Comment by ionutvmi** [comment]
you could do something like:
```
    "components": "${workspace}/src/components" // the root folder is now /src
```
if you don't have that many subfolders in /src.

I will look into how we can add support for redefining the root folder.

**Comment by ionutvmi** [comment]
We now have support for this in 1.7.0

    "$root": "${workspace}/src" // the root folder is now /src

**Comment by justindarner** [comment]
You rock! That solution seems to work well. 

The autocomplete kicks in at this point:
`
import Item from 'common/
`
Using this setting
```
{
  "path-autocomplete.pathMappings": {
    "common": "${workspace}/app/common",
    "pages": "${workspace}/app/pages"
  }
}
```

**Comment by goldo** [comment]
@ionutvmi it's not working for me im at v `1.31.1`

**Comment by ionutvmi** [comment]
Hi @goldo ,

Can you post your configuration and an example folder structure ?
I will try to reproduce the issue

**Comment by goldo** [comment]
simple create-react-app structure:
```
/
    src/
        components/
        routes /
```
and I want to import like this:
`import Button from 'components/Button'`

The only way I found was adding 
`    "components": "${workspace}/src/components", ` 
for each folder inside `src`.

I simply couldn't target `$root` as `src`

**Comment by ionutvmi** [comment]
@goldo , I was able to replicate the issue
Currently looking into it.

**Comment by ionutvmi** [comment]
@goldo this should be fixed on 1.13.2

**Comment by goldo** [comment]
@ionutvmi works like a charm, thanks
